server/index.js

```diff
@@ -57,23 +66,33 @@ io.on('connection', (socket) => {
         false
       ).catch(console.error),
     ])
-      .then(([translationObj, transciptionObj]) => {
-        const sessionRecord = {
-          user: socket.id,
-          langSource: data.langSource,
-          langTarget: data.langTarget,
-          ...translationObj,
-          ...transciptionObj,
-          convertElapsedTime: conversionTime - receivedTime,
-          serverElapsedTime: Date.now() - receivedTime,
-        };
-
-        console.log(JSON.stringify(sessionRecord));
-      })
-      .then(() => socket.emit('session-complete'))
-      .catch(() => {
-        console.error;
-        socket.emit('session-error');
-      });
+    .then((resp) => {
+      socket.emit('session-complete')
+      console.log('    Translation & Transcription complete')
+      return resp
+    })
```

Comment:
@blakebequette What is resp here? What is the parameter 'resp' in this call-back function?

Reply:
I'm just passing the array [translated, transcribed] to the next function. I put this here so that it closes the client socket before it does all the session logging stuff